protoc-gen-swagger/genswagger/template.go

```diff
@@ -1599,6 +1599,9 @@ func updateSwaggerObjectFromJSONSchema(s *swaggerSchemaObject, j *swagger_option
 	s.MaxProperties = j.GetMaxProperties()
 	s.MinProperties = j.GetMinProperties()
 	s.Required = j.GetRequired()
+    if overrideType := j.GetType(); len(overrideType) == 1 {
```

Comment:
Is this file gofmt'ed? It looks out of place in this diff.

Reply:
I assumed VSCode was formatting it right. Apparently not.. This has been fixed.